src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerTeamController.cs

```diff
@@ -562,6 +595,12 @@ public ActionResult CheckFunding(AccountDashboardViewModel model)
             return PartialView(model);
         }
 
+        [ChildActionOnly]
+        public ActionResult ContinueSetupForSingleReservation(AccountDashboardViewModel model)
+        {
+            return PartialView(model);
```

Comment:
Could pass the reservation in as the viewmodel here***

Reply:
Done